packages/cxl-ui/src/components/cxl-light-card.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import cxlLightCardStyles from '../styles/cxl-light-card-css.js';
+
+@customElement('cxl-light-card')
+export class CXLCourseCardElement extends LitElement {
+  static get styles() {
+    return [cxlLightCardStyles];
+  }
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) theme = 'light-card';
+
+  @property({ type: String }) title = 'Account based marketing';
+
+  @property({ type: String }) time = '3h 00min';
+
+  @property({ type: String }) instructorPrefix = 'By:';
+
+  @property({ type: String }) instructor = 'Ton Wesseling';
+
+  @property({ type: String }) avatar =
+    'https://cxl.com/institute/wp-content/uploads/2020/05/48192546_10156982340630746_8127333122065825792_n-wpv_400pxx400px_center_center.jpg';
+
+  @property({ type: Boolean }) hasBadges = false;
+
+  render() {
+    return html`
+      <div class="container">
+        <header class="entry-header">
+          <vaadin-icon icon="cxl:new"></vaadin-icon>
+          <div>
+            <div class="entry-title">
+              ${this.title}
+              <vaadin-icon icon="lumo:checkmark"></vaadin-icon>
+            </div>
+            <div class="entry-byline">
+              <div><vaadin-icon icon="lumo:clock"></vaadin-icon>${this.time}</div>
+              <div>${this.instructorPrefix}&nbsp;${this.instructor}</div>
+            </div>
+            <slot name="badges"></slot>
```

Comment:
> How should I pass the badges data? Do we know how will the different roadmaps themes be defined?

I think we should rename it to `footer` and let us pass any HTML we want. It will be more flexible and easier to manage. Today requirement is for badges but it might be also text tags etc.